main.tf

```diff
@@ -1,101 +1,80 @@
-# Configure the Google Cloud Provider
-
-terraform {
-  required_providers {
-    google = {
-      source  = "hashicorp/google"
-      version = "~> 4.85.0" 
-    }
-  }
-}
-
-provider "google" {

```

Comment:
removed to allow terraform to use implicit default provider config which would be
```
provider "google" {}
```

and provide project and region to resources (and also allow resources to be created in multiple regions (subnets, vms)